ui/src/modules/Metrics/Circles/History/CircleRow.tsx

```diff
@@ -14,27 +14,30 @@
  * limitations under the License.
  */
 
-import React, { useState } from 'react';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
 import Text from 'core/components/Text';
 import Styled from './styled';
 import { getStatus } from '../../helpers';
-import CircleReleasesTable from './CircleReleasesTable';
 import { CircleHistory } from '../interfaces';
+import routes from 'core/constants/routes';
 import { humanizeDateFromSeconds, dateTimeFormatter } from 'core/utils/date';
 
 type Props = {
   circle: CircleHistory;
 };
 
 const CircleRow = ({ circle }: Props) => {
-  const [activeRow, setActiveRow] = useState(false);
+  const history = useHistory();
 
   return (
     <Styled.CircleRow>
       <Styled.StatusLine status={getStatus(circle?.status)} />
       <Styled.TableRow
-        onClick={() => setActiveRow(!activeRow)}
         data-testid={`circle-row-${circle.id}`}
+        onClick={() =>
+          history.push(`${routes.circlesComparation}?circle=${circle.id}`)
```

Comment:
In `core/utils/path` we have the `addParam` method to handle this kind of route.
```suggestion
          addParam('circle', routes.circlesComparation, history, id)
```

Reply:
i didn't know, done